client/my-sites/theme/main.jsx

```diff
@@ -766,11 +759,7 @@ class ThemeSheet extends Component {
 		const placeholder = <span className="theme__sheet-placeholder">loading.....</span>;
 		const title = name || placeholder;
 		const tag = author ? translate( 'by %(author)s', { args: { author: author } } ) : placeholder;
-		const shouldRenderButton =
-			! retired &&
-			! isWPForTeamsSite &&
-			! this.shouldRenderForStaging() &&
-			( ! this.hasWpOrgThemeUpsellBanner() || ! isLoggedIn );
+		const shouldRenderButton = ! retired && ! isWPForTeamsSite && ! this.shouldRenderForStaging();
```

Comment:
Im confused why the isLoggedIn check was tied to `hasWpOrgThemeUpsellBanner` in the first place. This implies the check for upsell banner was true while logged out?

This part of the check is why no CTA was showing for community themes.

Reply:
It looks like the intent was no button for free/personal/premium and activate for logged-out (linking to log-in rather than activate) See #78537.

This was fine when we had the banner, but not afterwards.